modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsRepository.java

```diff
@@ -320,4 +347,51 @@ public void saveObsolescenceInfo(StatisticsKey key) {
             }
         });
     }
+
+    /**
+     * Subscribe to all local statistics changes.
+     *
+     * @param subscriber Local statitics subscriber.
+     */
+    public void subscribeToLocalStatistics(
+        Consumer<GridTuple3<StatisticsKey, ObjectStatisticsImpl, AffinityTopologyVersion>> subscriber
```

Comment:
let's introduce class with meaningful name instead of GridTuple3

Reply:
Replaced with new ObjectStatisticsEvent.